src/lighteval/metrics/metrics_sample.py

```diff
@@ -161,10 +159,8 @@ def __init__(
                 Defaults to None if no normalization is applied.
             strip_strings (bool, optional): Whether to strip both reference and predictions. Defaults to False.
         """
-        if aggregation_function is None:
-            aggregation_function = max
-        self.aggregation_function = aggregation_function
 
+        self.aggregation_function = aggregation_function
```

Comment:
The aggregation function can also be passed as None accidentally by the user, we want to catch that